pkgs/games/prismlauncher/wrapper.nix

```diff
@@ -40,10 +46,66 @@ symlinkJoin {
   ]
   ++ lib.optional (lib.versionAtLeast qtbase.version "6") qtwayland;
 
-  postBuild = ''
-    wrapQtAppsHook
+  # Passthrough
+  # Ref: https://github.com/NixOS/nixpkgs/blob/5e871d8aa6f57cc8e0dc087d1c5013f6e212b4ce/pkgs/build-support/build-fhsenv-bubblewrap/default.nix#L170
+  wrapperPreExec = lib.optionalString enableBubblewrap ''
+    args=()
+    if [[ "$DISPLAY" == :* ]]; then
+        local_socket="/tmp/.X11-unix/X''${DISPLAY#?}"
+        args+=(--ro-bind-try "$local_socket" "$local_socket")
+    fi
+    if [[ "$WAYLAND_DISPLAY" = /* ]]; then
+        args+=(--ro-bind-try "$WAYLAND_DISPLAY" "$WAYLAND_DISPLAY")
+    elif [[ -n "$WAYLAND_DISPLAY" ]]; then
+        args+=(--ro-bind-try "$XDG_RUNTIME_DIR/$WAYLAND_DISPLAY" "/tmp/$WAYLAND_DISPLAY")
+    fi
   '';
 
+  bwrapArgs = lib.optionals enableBubblewrap [
+    "--unshare-user"
+    "--unshare-ipc"
+    "--unshare-pid"
+    "--unshare-uts"
+    "--unshare-cgroup"
+    "--die-with-parent"
+
+    "--dev /dev"
+    "--proc /proc"
+    "--ro-bind /nix /nix"
+    "--ro-bind /etc /etc"
+    "--tmpfs /tmp"
+
+    # Network is required.
+    "--share-net"
+    "--ro-bind /run/systemd/resolve /run/systemd/resolve"
+
+    # Mesa & OpenGL.
+    "--ro-bind /run/opengl-driver /run/opengl-driver"
+    "--dev-bind-try /dev/dri /dev/dri"
+    "--ro-bind-try /sys/class /sys/class"
+    "--ro-bind-try /sys/dev/char /sys/dev/char"
+    "--ro-bind-try /sys/devices/pci0000:00 /sys/devices/pci0000:00"
+    "--ro-bind-try /sys/devices/system/cpu /sys/devices/system/cpu"
+
+    # Audio.
+    "--setenv XDG_RUNTIME_DIR /tmp"
+    ''--ro-bind-try "$XDG_RUNTIME_DIR/pulse" /tmp/pulse''
+    ''--ro-bind-try "$XDG_RUNTIME_DIR/pipewire-0" /tmp/pipewire-0''
+
+    # Runtime args from `wrapperPreExec`.
+    ''"''${args[@]}"''
+
+    # Data storage.
+    ''--bind "''${XDG_DATA_HOME:+$HOME/.local/share}/PrismLauncher" $HOME/.local/share/PrismLauncher''
```

Comment:
This should be a `-$HOME/…`, not a `+`.
Otherwise it expands to `--bind /PrismLauncher /home/…/.local/share/PrismLauncher` if `XDG_DATA_HOME` is not set.

Reply:
Oops, fixed now. Bash is so hard.